recipes/miniaudio/all/conanfile.py

```diff
@@ -12,46 +14,85 @@ class MiniaudioConan(ConanFile):
     license = ["Unlicense", "MIT-0"]
     settings = "os", "compiler", "build_type", "arch"
 
-    no_copy_source = True
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "header_only": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "header_only": False,
```

Comment:
```suggestion
        "header_only": True,
```
The upstream provides this as header-only, keep it by default, users expect the same behavior from the upstream.
Also, the CI has a feature which will build both header-only and non header-only packages.